ouroboros-consensus/src/Ouroboros/Consensus/Node/Exit.hs

```diff
@@ -10,11 +14,15 @@ module Ouroboros.Consensus.Node.Exit
   , toExitReason
   ) where
 
+import           Data.Proxy (Proxy)
+import           Data.Typeable (Typeable)
+
 import           Control.Exception (AsyncException (..), SomeException,
                      fromException)
 
 import           Control.Monad.Class.MonadAsync (ExceptionInLinkedThread (..))
 
+import           Ouroboros.Consensus.Block (StandardHash)
```

Comment:
Convention: separate `Ouroboros.Consensus` imports from `Ouroboros.Consensus.Storage` imports. (In that order)